astro/src/content/docs/get-started/core-concepts/roles.mdx

```diff
@@ -99,7 +99,7 @@ _FusionAuth application roles_
 | `connector_manager`        | `631ecd9d-8d40-4c13-8277-80cedb823701` | Can add and edit Connectors. <AvailableSince since="1.18.0"inline={true}/>                                                                                                     |
 | `consent_deleter`          | `631ecd9d-8d40-4c13-8277-80cedb8236fc` | Can delete consents.                                                                                                                                                           |
 | `consent_manager`          | `631ecd9d-8d40-4c13-8277-80cedb8236fd` | Can add and edit consents.                                                                                                                                                     |
-| `email_template_manager`   | `631ecd9d-8d40-4c13-8277-80cedb8236e7` | Can add and edit email templates.                                                                                                                                              |
+| `email_template_manager`   | `631ecd9d-8d40-4c13-8277-80cedb8236e7` | Can add, edit and delete email templates.                                                                                                                                      |
```

Comment:
I believe `entity_manager` and `webhook_manager` need the same addition.

Reply:
hrmm. Will expand this doc change, then.